ports/wampcc/vcpkg.json

```
@@ -1,19 +1,26 @@
 {
   "name": "wampcc",
-  "version-string": "2019-09-04",
-  "port-version": 2,
+  "version-date": "2019-09-04",
+  "port-version": 3,
```

```suggestion
  "version-date": "2019-09-04",
```

Since the version scheme change makes the versions not comparable we should reset `port-version` back to 0.

---

Hm in the past the convention was to not reset the `port-version` because the version is still the same.

---

@vicroms We should not remove `port-version` since the real version is not changed.